tools/fileinfo/features/pe-packers/test.py

```diff
@@ -1,5 +1,31 @@
 from regression_tests import *
 
+class Test_ActiveMark(Test):
+    settings=TestSettings(
+        tool='fileinfo',
+        input=[
+            'sample_activemark.dat',
+        ],
+        args='--json'
+    )
+
+    def test_corrupted_pe(self):
+        assert self.fileinfo.succeeded
+        self.assertTrue((self.fileinfo.output['tools'][1]['name'] == 'ActiveMark'))
```

Comment:
Nitpick: Redundant brackets around the argument to `self.assertTrue()`. For example, in `tools/fileinfo/features/vmprotect-heuristic/test.py`, you have a similar statement without those brackets:
```python
self.assertEqual(self.fileinfo.output['tools'][0]['name'], 'Petite')
```
Consider removing them.